workflows/edit_data_packages/update_a_package.Rmd

```diff
@@ -1,50 +1,36 @@
 ## Update a package
 
-To update a package, use `arcticdatautils::publish_update`. This function has an argument for adding data `PID`s (or otherwise including existing data `PID`s to make sure that they stay with the package). This function allows you to add or remove data objects, and/or make metadata edits.
+Once you have updated the data objects and saved the metadata to a file, we can update the metadata and add the new `pid` to the resource map using `publish_update`.
 
-First make sure you have the package you want to update, [loaded into R](#get-package-and-eml).
+To update a package, use `arcticdatautils::publish_update`. This function has an argument for adding data `PID`s (or otherwise including existing data `PID`s) to make sure that they stay with the package. This function allows you to add or remove data objects, and/or make metadata edits.
 
-### Updating metadata
+First make sure you have the package you want to update, [loaded into R](#get-package-and-eml) using `get_package`.
 
-If you need to make edits to the metadata, do so. Then write the `EML` metadata to a file.
+### Publish update
 
-``` {r, eval = F}
-eml_path <- "path/to/metadata/science_metadata.xml"
-write_eml(eml, eml_path)
-```
+Now we can update your data package.
 
-Next verify the eml is valid.
+```{r, eval = F}
+eml_path <- "path/to/your/saved/eml.xml"
 
-``` {r, eval = F}
-EML::eml_validate(eml_path)
```

Comment:
Is this discussed later in the training?

Reply:
it was discussed previously, but those lines were removed not added

Reply:
Yep, saw that they were removed and just wanted to make sure we had it in there somewhere.